types/user/user.ts

```diff
@@ -0,0 +1,11 @@
+export interface User {
+  user_id: string;
```

Comment:
I think the user_id will be a primary key that will be incrementing as an integer. I'm thinking we should type it as an integer?
```suggestion
  user_id: number;
```

Reply:
Wouldn't we use UUID for user_id though, so it should be a string? 🤔

Reply:
Yep you're right. I asked this question a few days ago and I was under the impression it would be a number, but UUID is better.